packages/syft/src/syft/store/blob_storage/seaweedfs.py

```diff
@@ -42,26 +42,16 @@
 from ...types.syft_object import SYFT_OBJECT_VERSION_1
 from ...types.uid import UID
 from ...util.constants import DEFAULT_TIMEOUT
-from ...util.telemetry import TRACING_ENABLED
-
-logger = logging.getLogger(__name__)
+from ...util.telemetry import instrument_botocore
 
 MAX_QUEUE_SIZE = 100
 WRITE_EXPIRATION_TIME = 900  # seconds
 DEFAULT_FILE_PART_SIZE = 1024**3  # 1GB
 DEFAULT_UPLOAD_CHUNK_SIZE = 1024 * 800  # 800KB
 
-if TRACING_ENABLED:
-    try:
-        # third party
-        from opentelemetry.instrumentation.botocore import BotocoreInstrumentor
-
-        BotocoreInstrumentor().instrument()
-        message = "> Added OTEL BotocoreInstrumentor"
-        print(message)
-        logger.info(message)
-    except Exception:  # nosec
-        pass
+logger = logging.getLogger(__name__)
+
+instrument_botocore()
```

Comment:
Q: Instead of calling library specific instrumentation, at the specific locations, could we not call all of them during FastAPI startup?

Reply:
no let it load in the order of module imports.